core/primitives/src/block.rs

```diff
@@ -26,6 +26,10 @@ pub struct BlockHeaderInner {
     pub prev_state_root: MerkleHash,
     /// Root hash of the transactions in the given block.
     pub tx_root: MerkleHash,
+    /// Root hash of the chunk receipts in the given block.
+    pub chunk_receipts_root: MerkleHash,
```

Comment:
Check the validity of the two new roots in `ChainUpdate::process_block`?
And add the same check for `tx_root`, it seems to be missing.